analyzers/tests/SonarAnalyzer.Test/Rules/AvoidExcessiveClassCouplingTest.cs

```diff
@@ -87,53 +87,57 @@ public void Foo(IList<ICollection<IEnumerable<IComparer<Stack<Queue<int>>>>>> di
 
         [TestMethod]
         public void AvoidExcessiveClassCoupling_Task_Not_Counted() =>
-            withThreshold0.AddSnippet(@"
-using System.Threading.Tasks;
-public class Tasks // Compliant, Task types are not counted
-{
-    public void Foo<T>(Task task1, Task<T> task2) { }
-}
-").Verify();
+            withThreshold0.AddSnippet("""
+                using System.Threading.Tasks;
+                public class Tasks // Compliant, Task types are not counted
+                {
+                    public void Foo<T>(Task task1, Task<T> task2) { }
+                }
+                """).VerifyNoIssues();
```

Comment:
The whole file is aligning the snippet in the first column, so if we really want to change it I would go for the entire file to keep consistency

Reply:
(I would keep the old alignment)